bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx

```diff
@@ -57,6 +57,9 @@ export default class DropdownListItem extends Component {
         {
           children || this.renderDefault()
         }
+        {
+          label === 'Set Status' ? <Icon iconName="right_arrow" key="icon" className={styles.moreIcon} /> : null
```

Comment:
this will not works when you change the language.

Reply:
Make use of the `children` prop for cases like this. The dropdown list item can be anything you dont necessarily need a Icon + Label thats just the default render.